roles/teslamate/defaults/main.yml

```diff
@@ -0,0 +1,158 @@
+#########################################################################
+# Title:            Sandbox: teslamate                                  #
+# Author(s):        maindust                                            #
+# URL:              https://github.com/saltyorg/Sandbox                 #
+# --                                                                    #
+#########################################################################
+#                   GNU General Public License v3.0                     #
+#########################################################################
+---
+# defaults/main.yml
+
+teslamate_name: teslamate
+
+################################
+# Settings
+################################
+
+TM_DB_USER: "{{ user.name }}"
+TM_DB_PASS: "password4321"
+TM_DB_NAME: "teslamate"
+TESLAMATE_SECRET_KEY: "{{ teslamate_saltbox_facts.facts.secret_key }}"
+
+################################
+# Paths
+################################
+
+teslamate_paths_folder: "{{ teslamate_name }}"
+teslamate_paths_location: "{{ server_appdata_path }}/{{ teslamate_paths_folder }}"
+teslamate_paths_folders_list:
+  - "{{ teslamate_paths_location }}"
+
+################################
+# Web
+################################
+
+teslamate_web_subdomain: "{{ teslamate_name }}"
+teslamate_web_domain: "{{ user.domain }}"
+teslamate_web_port: "4000"
+teslamate_web_url: "{{ 'https://' + (teslamate_web_subdomain + '.' + teslamate_web_domain
+                  if (teslamate_web_subdomain | length > 0)
+                  else teslamate_web_domain) }}"
+
+################################
+# DNS
+################################
+
+teslamate_dns_record: "{{ teslamate_web_subdomain }}"
+teslamate_dns_zone: "{{ teslamate_web_domain }}"
+teslamate_dns_proxy: "{{ dns.proxied }}"
+
+################################
+# Traefik
+################################
+
+teslamate_traefik_sso_middleware: ""
+teslamate_traefik_middleware_default: "{{ traefik_default_middleware }}"
```

Comment:
Confirming the appointment has its own auth and therefore disabling Authelia/SSO by default is appropriate

Reply:
Added traefik sso middleware to app. Thanks for the catch